internal/distributed/proxy/service.go

```diff
@@ -132,6 +133,10 @@ func authenticate(c *gin.Context) {
 			return
 		}
 	}
+	if !proxy.Params.CommonCfg.AuthorizationEnabled {
```

Comment:
1. Why need to set root as a default user if authorization is disabled? 
2.  is it reasonable to set the current name to root after parsing the user name successfully if authorization is disabled